src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -738,42 +730,42 @@ void ApplicationManagerImpl::OnHMIStartedCooperation() {
   hmi_cooperating_ = true;
   MessageHelper::SendGetSystemInfoRequest(*this);
 
-  std::shared_ptr<smart_objects::SmartObject> is_vr_ready(
+  utils::SharedPtr<smart_objects::SmartObject> is_vr_ready(
       MessageHelper::CreateModuleInfoSO(hmi_apis::FunctionID::VR_IsReady,
                                         *this));
   rpc_service_->ManageHMICommand(is_vr_ready);
 
-  std::shared_ptr<smart_objects::SmartObject> is_tts_ready(
+  utils::SharedPtr<smart_objects::SmartObject> is_tts_ready(
       MessageHelper::CreateModuleInfoSO(hmi_apis::FunctionID::TTS_IsReady,
                                         *this));
   rpc_service_->ManageHMICommand(is_tts_ready);
 
-  std::shared_ptr<smart_objects::SmartObject> is_ui_ready(
+  utils::SharedPtr<smart_objects::SmartObject> is_ui_ready(
       MessageHelper::CreateModuleInfoSO(hmi_apis::FunctionID::UI_IsReady,
                                         *this));
   rpc_service_->ManageHMICommand(is_ui_ready);
 
-  std::shared_ptr<smart_objects::SmartObject> is_navi_ready(
+  utils::SharedPtr<smart_objects::SmartObject> is_navi_ready(
       MessageHelper::CreateModuleInfoSO(
           hmi_apis::FunctionID::Navigation_IsReady, *this));
   rpc_service_->ManageHMICommand(is_navi_ready);
 
-  std::shared_ptr<smart_objects::SmartObject> is_ivi_ready(
+  utils::SharedPtr<smart_objects::SmartObject> is_ivi_ready(
       MessageHelper::CreateModuleInfoSO(
           hmi_apis::FunctionID::VehicleInfo_IsReady, *this));
   rpc_service_->ManageHMICommand(is_ivi_ready);
 
-  std::shared_ptr<smart_objects::SmartObject> is_rc_ready(
+  utils::SharedPtr<smart_objects::SmartObject> is_rc_ready(
       MessageHelper::CreateModuleInfoSO(hmi_apis::FunctionID::RC_IsReady,
                                         *this));
   rpc_service_->ManageHMICommand(is_rc_ready);
 
-  std::shared_ptr<smart_objects::SmartObject> button_capabilities(
+  utils::SharedPtr<smart_objects::SmartObject> button_capabilities(
       MessageHelper::CreateModuleInfoSO(
           hmi_apis::FunctionID::Buttons_GetCapabilities, *this));
   rpc_service_->ManageHMICommand(button_capabilities);
 
-  std::shared_ptr<smart_objects::SmartObject> mixing_audio_supported_request(
+  utils::SharedPtr<smart_objects::SmartObject> mixing_audio_supported_request(
```

Comment:
@Ypostolov Check all these changes. We already use shared_ptr from C++11 . Return previous code please